docs/plugins/embedart.rst

```diff
@@ -91,6 +91,9 @@ embedded album art:
   its own currently associated album art. The command prompts for confirmation
   before making the change unless you specify the ``-y`` (``--yes``) option.
 
+* ``beet embedart [-u IMAGE_URL] QUERY``: embed image specified in the URL
```

Comment:
Could you do me a favor please and fix that typo in the `beed embedart -f` help text. Unrelated with this PR but since we are at it, doesnt hurt I guess.

embed images into ~the~ every track

Thanks a lot!

Reply:
Done